authority/provisioner/sign_options.go

```diff
@@ -443,7 +444,18 @@ func newProvisionerExtensionOption(typ Type, name, credentialID string, keyValue
 	}
 }
 
+// WithControllerOptions returns the provisionerExtensionOption options from the
+// controller. Currently only the claim DisableSmallstepExtensions is used.
```

Comment:
```suggestion
// WithControllerOptions updates the provisionerExtensionOption with options from the
// controller. Currently only the DisableSmallstepExtensions claim is used.
```

Reply:
It can be a bit confusing when both options and claims are used to refer to the same / similar thing(s). I know both have been around for a while, but maybe we need to (start) set(ting) things straight now. 

The `DisableSmallstepExtensions` boolean sounds like a plain option to me. Claims, to me, are things that end up in JWTs. Is this the proper distinction to make?

Reply:
Fixed with 3790bd01, I've changed "claim" with "provisioner claim" in that comment.